### PR TITLE
[Bug15590] Prevent memory leak (and an eventual crash) when cropping an image

### PR DESCRIPTION
**Pull request by livecodepanos**

### CODE REVIEW

File: engine/src/iutil.cpp [hunk 1]
```diff
@@ -464,8 +464,14 @@ void MCImage::crop(MCRectangle *newrect)
 	MCImageBitmapCopyRegionToBitmap(t_bitmap, t_cropimage, xorigin, yorigin, trect.x, trect.y, trect.width, trect.height);
 
 	unlockbitmap(t_bitmap);
+	
+	// PM-2015-07-13: [[ Bug 15590 ]] Fix memory leak
+	MCImageFreeBitmap(t_bitmap);
```
Comment (livecodeali) [inline engine/src/iutil.cpp:467]: The image bitmap pointed to by `t_bitmap` shouldn't need to be freed, as it will be freed by `unlockbitmap`

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 085060d

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 085060d4254cdc1bd77a44ad07724c4fdc18650a

**Comment by livecode-vulcan** [comment]
😎 test success 085060d4254cdc1bd77a44ad07724c4fdc18650a
- try-linux-x86: [success](https://vulcan.livecode.com/builders/try-linux-x86/builds/498)
- try-win-x86: [success](https://vulcan.livecode.com/builders/try-win-x86/builds/556)
- try-ios: [success](https://vulcan.livecode.com/builders/try-ios/builds/444)
- try-android-armv6: [success](https://vulcan.livecode.com/builders/try-android-armv6/builds/509)
- try-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-linux-x86_64/builds/444)
- try-mac: [success](https://vulcan.livecode.com/builders/try-mac/builds/435)

**Pull request merged**